### DISCUSSION TITLE
Ride Stat Calculator Toggle

### DISCUSSION THREAD

**Discussion by Ballpitwarrior** [OpenRCT2/OpenRCT2#19689] [Feature requests]
Hello-

Would it be possible to add maybe as a cheat menu and then later onto ride windows a way to manage ride stat calculation by ride ID?  Sometimes shoestringing or decorative vehicles block stat calculation which can mean you have to creatively get around it, but if it was a toggle to make a ride not “calculate-able” to the engine then it could bypass this problem to let it know that ride doesn’t need to have stats calculated.  For shoestrung rides, this could mean the stats are inputted manually in cases where it has an issue and don’t want to spend the time to correct it.

**Comment by ocalhoun6** [comment]
#19683 should help with this, once it's done.

Then, as long as you have less than 4 shoestring/decorative rides, stats will still be calculated.

Though, personally, I think that #19532 would be the ultimate solution.
